runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -363,21 +363,88 @@ class WorkItemListPage {
   }
 
   /* Workitem filter pulldown */
-  get workItemFilterPulldown () {
-//    return element(by.css(".dropdown.filter-dropdown"));
-    return element(by.css(".filter-option.pull-left"));
+  get workItemFilterFieldsPulldown () {
+    return element(by.css(".filter-fields.dropdown-toggle"));
   }
-  clickWorkItemFilterPulldown () {
-    return this.workItemFilterPulldown.click();
+  clickWorkItemFilterFieldsPulldown () {
+    return this.workItemFilterFieldsPulldown.click();
   }
+
+  get filterByAssignee () {
+    browser.wait(until.presenceOf(element(by.xpath("//li[2]/a[@class='filter-field dropdown-item']"))), constants.WAIT, 'Failed to filter by Assignee Type');
```

Comment:
xpaths are bad. These will be fixed as part of the xpath removal effort, which is pending.